chsdi/tests/e2e/test_links.py

```diff
@@ -8,12 +8,14 @@
 class TestLinks(TestsBase):
 
     def test_external_links(self):
-        h = httplib2.Http(timeout=10)
+        headers = {
+            'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/37.0.2062.68 Safari/537.36'
+        }
         for i in range(23):
             response = self.testapp.get('/rest/services/ech/MapServer/ch.kantone.cadastralwebmap-farbe/%d/htmlPopup' % i, status=200)
 
             soup = response.html
             for a in soup.findAll('a'):
                 link = a.get('href')
-                resp, content = h.request(link, headers={'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/37.0.2062.68 Safari/537.36'})
-                self.assertTrue(resp.status == 200, link)
+                r = requests.get(link, timeout=10, headers=headers)
+                self.assertTrue(r.status_code == 200, link)
```

Comment:
The `link` part will output the link that failed on failure. It's for readability in case of failure.